local/kaltura/service.php

```diff
@@ -90,6 +90,7 @@
 
 $PAGE->set_url($serviceurl);
 $PAGE->set_context(context_system::instance());
+$previewltilaunchurl = new moodle_url('/local/kaltura/bsepreview_ltilaunch.php?playurl=' . urlencode($url));
```

Comment:
i usually do camel casing, however, if you look around Moodle's code (and our plugin's as well) you'd notice that the code is not camel cased, for the most part. it's more of a C-style code. so i kept on with it.